client/src/plus/deep-dives/article.tsx

```diff
@@ -94,7 +93,6 @@ export default function Article({ slug }: { slug: string }) {
           />
         </div>
       </div>
-      <Feature />
       <Survey slug={slug} />
       <div
         className={`deep-dive-article-footer ${
```

Comment:
Sorry, I don't know where to add this, can you increase the padding for `.deep-dive-article-footer a {
    display: block;
    padding: 48px 24px;
}` like this for desktop